app/webpack/stylesheets/helpers.scss

```diff
@@ -23,6 +23,13 @@
   .only-print, .only-print * {
     display: inherit !important;
   }
+  .pagebreak {
+    page-break-before: always;
+  }
```

Comment:
As the `page-break-before` property is used by print media, I would suggest adding this to the file `app/webpack/stylesheets/print-styles.scss`

Reply:
@naseberry thanks. Following your comment I spotted that there was already existing page_break_before css so I have added a commit to use that instead. I have moved the open-banking-guidance-img css to print-styles.scss, if that is OK?